src/js/clientApp.js

```diff
@@ -15,9 +15,11 @@ try {
 }
 
 import { VERSION } from "./versionInfo";
-
 console.log('appVersion ->', VERSION);
 
+import { objectFilter } from "./utils";
```

Comment:
but im using it everywhere in the application. and clientapp is like the mommy of the components